circus/tests/test_client.py

```diff
@@ -78,41 +77,51 @@ def test_client(self):
         yield self.stop_arbiter()
 
 
+TMP_FILE = './HOOK_CALLED'
+
+
 def long_hook(*args, **kw):
-    time.sleep(5)
+    with open(TMP_FILE, 'a'):
```

Comment:
I tried to do it without file but the way the hook is resolved and the lack of a way to pass arguments to the hook excludes any communication between the hook and the test. Even if we had a tempfile we wouldn't be able to communicate the file name/fd to the hook. 

I know it's a gross hack but I was unable to come up with something better without introducing things to the core.